include/pi-peps/linalg/arpack-rcdn.h

```diff
@@ -8,6 +8,7 @@
 #include <cmath>
 #include <iostream>
 #include <vector>
+#include <complex>
 
 #ifndef BLASINT
```

Comment:
where is `BLASINT` defined? should we include a proper header?

Reply:
BLASINT is defined in ITensor lapack wrapper headers, which are included

Reply:
but not in this file, or am I wrong?